apps/app/src/styles.css

```diff
@@ -198,3 +198,7 @@ select {
 	--input: 240 3.7% 15.9%;
 	--ring: 142.4 71.8% 29.2%;
 }
+
+* {
+	border-color: hsl(var(--border));
+}
```

Comment:
@goetzrobin @elite-benni - just wanted to call special attention to this change. Im pretty sure I noticed this was missing in storybook and saw there was a similar issue in the docs too with the select component border. When I checked shadcn I found this was missing but might just want to double check all the components make sure no adverse effects between dark/light modes. At quick glance everything seems fine